core/src/main/scala/com/nvidia/spark/rapids/tool/tuning/AutoTuner.scala

```diff
@@ -657,11 +657,27 @@ class AutoTuner(
   def calculateJobLevelRecommendations(): Unit = {
     // TODO - do we do anything with 200 shuffle partitions or maybe if its close
     // set the Spark config  spark.shuffle.sort.bypassMergeThreshold
+    val isDynamicAllocationEnabled = appInfoProvider.getSparkProperty(
+      "spark.dynamicAllocation.enabled").contains("true")
     getShuffleManagerClassName match {
-      case Right(smClassName) => appendRecommendation("spark.shuffle.manager", smClassName)
-      case Left(comment) => appendComment("spark.shuffle.manager", comment)
+      case Right(smClassName) =>
```

Comment:
For the issue:
> Avoid recommend RapidsShuffleManager when dynamic executor allocation is on

Can we simplify this logic as :

```
if (isDynamicAllocationEnabled) {
  appendComment(dynamicAllocShuffleManagerComment)
} else {
 // existing code
}
```

Reply:
Additionally based on the issue description
> It would be great to allow skip shuffle manager configuration where it's configured as Celeborn or RSS shuffle manager in its user configuration.

Is checking for `isDynamicAllocationEnabled` sufficient or should we also check if the user has manually set custom external shuffle manager?

Reply:
> Is checking for isDynamicAllocationEnabled sufficient
Yes. there're more cases FYI. https://spark.apache.org/docs/latest/configuration.html#dynamic-allocation

> should we also check if the user has manually set custom external shuffle manager?
Do we have a way to parse the configurations from user? Typically, it's inherited configuration from customer's auto-tuner or even some end-user configurations.

Reply:
> Do we have a way to parse the configurations from user? Typically, it's inherited configuration from customer's auto-tuner or even some end-user configurations.

Yes, we can use `getPropertyValue("spark.shuffle.manager")` to get the value of existing shuffle manager from the original spark job

Reply:
Discussed with Ferdinand and launched a thread in Slack as it touches some customer info.